### ISSUE TITLE
Unchecking top tree item for IDE version still does compile/installation for that IDE

### ISSUE THREAD

**Issue by skydvrz** [Delphier/DxAutoInstaller#8]
It you uncheck the checkbox found just below the IDE version then the installer still deletes the BPLs for the disabled IDEs and recompiles.  If nothing else, this is super time-consuming if you have multiple IDE versions installed.  I have 6 IDE versions installed (really!).

**Comment by skydvrz** [comment]
Ok...  It turns out that this code needs to be remmed out.  It may be a hack and introduce other bugs but it does stop the installer from deinstalling/reinstalling components for unselected IDEs:

```
    procedure SetComponentNodeState(Node : TcxTreeListNode);
    begin
//        if not Node.Visible then
//            Exit;

````

**Comment by jeffovercash** [comment]
This is another variation on the "Default checkbox hides enabled settings" bug.  In app workaround for this is to uncheck the Hide Base components before unchecking that whole release.  With it checked the IDE Version checkbox does not uncheck the base components so the installer sees things to do still for that IDE.

**Comment by Delphier** [comment]
Fixed in 797f707
